liquid-compiler/src/block.rs

```diff
@@ -6,17 +6,17 @@ use super::TagBlock;
 use super::TagTokenIter;
 
```

Comment:
Please add a `BREAKING CHANGE` section to your commit message so it gets in the docs.

See https://www.conventionalcommits.org/en/v1.0.0/

Reply:
True, I'll add that on the final squash